lib/grpc_reflection/service/builder/util.ex

```diff
@@ -159,4 +159,54 @@ defmodule GrpcReflection.Service.Builder.Util do
 
   def trim_symbol("." <> symbol), do: symbol
   def trim_symbol(symbol), do: symbol
+
+  def proto_filename(module) do
```

Comment:
I don't think this is reliable when there are protobuf options like https://github.com/elixir-protobuf/protobuf?tab=readme-ov-file#one-file-per-module that can be triggered by users.  We would need to restrict them from using that option or find an alternative